### DISCUSSION TITLE
how to change the quickemu conf for cores and threads

### DISCUSSION THREAD

**Discussion by tanujv7** [quickemu-project/quickemu#1388] [Quickemu] [answered]
Hello guys, I have been trying to run the macos sonoma vm, but the default configuration quickemu uses, has only 2 threads and 2 cores, I want to change the cpu cores to 4, but not able to do so, even after changing the macos-sonoma.conf file, it always uses the default 2 cores

**Comment by lj3954** [comment, marked as answer]
Quickemu counts threads in the core count option. If you want to allocate 4 physical cores on a system with SMT/HT, set cpu_cores to 8 in your config file.

**Reply by tanujv7** [reply]
I see, thanks, I will try this 😊
